dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -412,7 +417,7 @@ public void doImport(PrintWriter out) throws IOException {
 
         for (File file : usersRolesXML) {
             try{
-				HibernateUtil.closeSession();
+				HibernateUtil.closeSession(); // todo: why this?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)